internal/environmentscleanup/service.go

```diff
@@ -292,30 +284,53 @@ func (s *Service) triggerEnvironmentDeprovisioning(instance internal.Instance) e
 	return nil
 }
 
-func (s *Service) cleanUpRuntimeCRs(runtimesToDelete []runtime, kebInstancesToDelete []internal.Instance) *multierror.Error {
-	kebInstanceExists := func(runtimeID string) bool {
-		for _, instance := range kebInstancesToDelete {
-			if instance.RuntimeID == runtimeID {
-				return true
-			}
-		}
+func (s *Service) cleanUpRuntimeCRs(runtimeCRs []runtime) error {
+	s.logger.Infof("RuntimeCRs to process: %+v", runtimeCRs)
 
-		return false
+	if len(runtimeCRs) == 0 {
```

Comment:
Is it really necessary? Would not `for` loop suffice?

Reply:
I will remove it.